### PR TITLE
[CLOUD-96] Use managed docker registry by default

### PR DESCRIPTION
**Pull request by maismail**
https://hopsworks.atlassian.net/browse/CLOUD-96
 
 still missing the Azure documentation but that should be done next week

### CODE REVIEW

File: docs/setup_installation/aws/restrictive_permissions.md [hunk 1]
```diff
@@ -269,6 +269,8 @@ Hopsworks put its logs in Amazon CloudWatch so that you can access them without
     }
 ```
 ### Upgrade permissions 
+!!! warning
```
Comment (berthoug) [inline docs/setup_installation/aws/restrictive_permissions.md:272]: I think we should just remove the section.

**Review by berthoug — CHANGES_REQUESTED** [review]

File: docs/setup_installation/aws/instance_profile_permissions.md [hunk 2]
```diff
@@ -31,6 +31,52 @@ Replace *BUCKET_NAME* with the appropriate S3 bucket name.
...
     },
+    {
+      "Sid": "AllowPullMainImages",
+      "Effect": "Allow",
+      "Action": [
+        "ecr:GetDownloadUrlForLayer",
+        "ecr:BatchGetImage"
+      ],
+      "Resource": [
+        "arn:aws:ecr:*:*:repository/filebeat",
```
[the remark this answers is not in the document]
Comment (maismail) [reply docs/setup_installation/aws/instance_profile_permissions.md:46]: yes, updated it

File: docs/setup_installation/aws/restrictive_permissions.md (hunk 1 above)
Comment (maismail) [reply docs/setup_installation/aws/restrictive_permissions.md]: done, removed

File: docs/setup_installation/azure/getting_started.md [hunk 3]
```diff
@@ -352,7 +352,46 @@ Click on *Save* (7).
...
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/azure/acr-retention.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/azure/acr-retention.png" alt="ACR retention policy">
+    </a>
+    <figcaption>ACR retention policy</figcaption>
+  </figure>
+</p>
+
+## Step 4: Adding a ssh key to your resource group
```
Comment (berthoug) [inline docs/setup_installation/azure/getting_started.md:394]: You need to update the links pointing to this.

**Review by berthoug — CHANGES_REQUESTED** [review]

File: docs/setup_installation/azure/getting_started.md (hunk 3 above)
Comment (maismail) [reply docs/setup_installation/azure/getting_started.md:394]: fixed

**Review by berthoug — APPROVED** [review]

**Pull request merged**